adapters/tpmn/tpmn_test.go

```diff
@@ -0,0 +1,20 @@
+package tpmn
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderTpmn, config.Adapter{
+		Endpoint: "http://example.com/pserver"}, config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})
```

Comment:
This is correct. But suggesting to match `Endpoint` url with `https://gat.tpmn.io/ortb/pbs_bidder` mentioned in bidder params

Reply:
update commit : 
[1390874](https://github.com/prebid/prebid-server/pull/2947/commits/1390874130016765f0134fe3a2321440c995f2dd)
[7896970](https://github.com/prebid/prebid-server/pull/2947/commits/7896970cb691e5e411608c3e012f1ffd32639757)